### PR TITLE
New icons, metadata, and ZIP

### PR DESCRIPTION
**Pull request by dburka**
Added a bunch of newly completed icons.

### CODE REVIEW

File: public/icons/meta-data.json [hunk 1]
```diff
@@ -6543,5 +6430,515 @@
...
+ {
+  "id": "ui_zoom_out",
+  "category": "symbols",
+  "path": "symbols/ui_zoom_out",
+  "tags": [
+   "Magnifying Glass",
+   "Navigation",
+   "User interface"
+  ],
+  "title": "UI Zoom In"
```
Comment (nburka) [inline public/icons/meta-data.json:6567]: should be "UI Zoom Out"

File: public/icons/meta-data.json [hunk 2]
```diff
@@ -6543,5 +6430,515 @@
...
+ {
+  "id": "ui_zoom",
+  "category": "symbols",
+  "path": "symbols/ui_zoom",
+  "tags": [
+   "Find",
+   "Magnifying Glass",
+   "Navigation",
+   "Search",
+   "User interface"
```
Comment (nburka) [inline public/icons/meta-data.json:6589]: should be "User Interface" for all of them (capital "I")

**Review by nburka — CHANGES_REQUESTED** [review]

File: public/icons/meta-data.json (hunk 2 above)
Comment (dburka) [reply public/icons/meta-data.json]: Good catch. Fixed the tags to be uppercase. There were a few more as well... I think I got them all.

File: public/icons/meta-data.json (hunk 1 above)
Comment (dburka) [reply public/icons/meta-data.json:6567]: Good catch. Fixed the zoom-in zoom-out icons.

**Pull request merged**